docs/spec/nsm-monitor.md

```diff
@@ -2,7 +2,7 @@
 Sidecar nsm-monitor
 ============================
 
-Specification
+Specification`
```

Comment:
Why the stray "`" character?

Reply:
- [ ] remove stray character in nsm-monitor.md